phpunit/tests/base.php

```diff
@@ -1,5 +1,17 @@
 <?php
+/**
+ * WordPress Importer plugin testcase.
+ *
+ * @package WordPress
```

Comment:
If two tags directly follow each other, shouldn't their values be aligned with each other ?

```php
/**
 * @package    WordPress
 * @subpackage Importer
 */
```

Reply:
Because currently WP Core doesn't align that:

Reply:
Hmm.. surprising.